mail_outplacement_report/i18n/sv_SE.po

```diff
@@ -0,0 +1,213 @@
+msgstr ""
+
+#. module: mail_outplacement_report
+#: model:mail.template,body_html:mail_outplacement_report.email_template_login_employee
+msgid "<div>\n"
+"                    <p>\n"
+"                        Hej ${object.name},<br/>\n"
+"                    </p>\n"
+"                    <p>\n"
+"                        Ett Konto har skapats och du kan nu logga på Digital Arbetsyta Fristående Aktör. Klicka på länken nedan för att logga in.<br/>\n"
+"                    </p>\n"
+"                        <div style=\"margin: 16px 0px 16px 0px;\">\n"
+"                            <a href=\"/web/login?${ctx['auth_login']}\" style=\"background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;\">\n"
+"                                Mitt konto\n"
+"                            </a>\n"
+"                        </div>\n"
+"                        <br/><br/>\n"
+"\n"
+"                        Med vänlig hälsning,<br/>\n"
+"                        ${object.company_id.name}<br/><br/>\n"
+"\n"
+"            <p>\n"
+"                    <hr/>\n"
+"                    <!-- FOOTER -->\n"
+"\n"
+"                    ${object.company_id.name}\n"
+"\n"
+"                    ${object.company_id.phone}\n"
+"                    % if object.company_id.email\n"
+"                    | <a href=\"'mailto:%s' % ${object.company_id.email}\" style=\"text-decoration:none; color: #454748;\">${object.company_id.email}</a>\n"
+"                    % endif\n"
+"                    % if object.company_id.website\n"
+"                    | <a href=\"'%s' % ${object.company_id.website}\" style=\"text-decoration:none; color: #454748;\">\n"
+"                    ${object.company_id.website}\n"
+"                </a>\n"
+"                    % endif\n"
+"            </p>\n"
+"\n"
+"                    </div>\n"
+"\n"
+"                "
+msgstr ""
+
+#. module: mail_outplacement_report
+#: model:mail.template,body_html:mail_outplacement_report.email_template_assigned_coach
+msgid "<div>\n"
+"                    <p>\n"
+"                        Hej ${object.partner_name or ''}!\n"
+"                    </p>\n"
+"                    <p>\n"
+"                        <strong>${object.employee_id.name} </strong>  är din handledare och du är välkommen till oss\n"
+"                        klockan ${object.performing_time  or ''} den ${object.performing_date or ''} på plats ${object.performing_operation_adress.street or ''},${object.performing_operation_adress.zip or ''} ${object.performing_operation_adress.city or ''}.\n"
+"                        Ta med dig en kopia på dina intyg, ID-handlingar och glatt humör.\n"
+"                    </p>\n"
+"                    <p>\n"
+"                        Anteckningsmaterial finns på plats.\n"
+"                    </p>\n"
+"                    <p>\n"
+"                        Hälsningar <br/>\n"
+"                        ${object.employee_id.address_id.name or ''}\n"
```

Comment:
not object.employee_id.name here?

Reply:
I have no idea really, this file was auto generated by Odoo. I suggest we try it and change if needed later.